src/scss/components/topnav/index.scss

```diff
@@ -37,11 +39,13 @@
 
```

Comment:
Please check the new specs for the branch icon.

Reply:
Changes have been done, please review.